k8s/aat/common/am/am-org-role-mapping-service.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: am-org-role-mapping-service
+  namespace: am
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: am-org-role-mapping-service
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: am-org-role-mapping-service
+    version: 0.0.1
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
```

Comment:
This is not needed if u are using latest java chart

Reply:
Hi Praveen, we will remove "useInterpodAntiAffinity: true"